### ISSUE TITLE
Practice Area Feedback - Name of Collab Touchpoint - Name of Project

### ISSUE THREAD

**Issue by allison0034** [department-of-veterans-affairs/va.gov-team#27629]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
- This looks great! I am uploading some UI issues, however Ryan has asked for a synchronous meeting to discuss some things not listed here.

## Feedback
- **Must** = the feedback must be applied
- **Should** = the feedback is best practice
- **Consider** = suggestions/enhancements

**Must:**
- Follow the typical pattern we use for ‘Sign in’. See [benefits application](https://design.va.gov/patterns/benefit-applications) or the schedule a healthcare appt. Use the green action link instead of the button.
- Remove the little blue line and ‘Online Tool'
- ![Screen Shot 2021-07-19 at 4.04.33 PM.png](https://images.zenhubusercontent.com/5f7e15543782a5c1ee10f1f0/b778045c-fd8a-4400-91de-b481d4a7843a)
- Do not use the wizard, we are trying to retire it. Use single screens
- Can we skip this question? We have all the bill details on this page.
- ![Screen Shot 2021-07-20 at 2.27.15 PM.png](https://images.zenhubusercontent.com/5f7e15543782a5c1ee10f1f0/19d1bcc9-19b4-4451-9cae-7d4f0b5c4680)

**Should:**

**Consider:**

## Platform directions
- Update "Issue Title"
- Link to collab cycle touchpoint meeting epic
- Add to the Feedback pipeline on the Platform Product Support board
- Add your feedback
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review**: VFS PM, yourself (optional)

**Comment by rebeccawalsh** [comment]
@allison0034 

Thanks! Answers below:

## Must
- We added this into our prototype for testing (which we just completed.) We've seen that the sign in / button / action link thing isn't working well; people ignore the sign in and don't see the button / action links. Because of this, we collaborated with Shawna and other VSA designers to reframe the action link to make it more obvious that the page has an interactive tool associated with it. We've heard from others that we avoid saying something is a 'tool' so that will be discussed, but it was very effective in testing from what we saw.
- See above
- The wizard already exists - it belongs to FSR. We're just adjusting a little bit of the content to route people. We're planning to make bigger changes and we're keeping Josh's suggestions in mind when we get around to it. This is not a priority for our team before release as it's already functional and we're on a timeline.
- We're looking into prefilling the initial question, however, this wizard doesn't exist solely for MCP, so we can't remove it. Bigger changes are planned, but not prior to initial release.

## Should

## Consider

I messaged Ryan about chatting sometime this afternoon or tomorrow, would you like to be on the invite if we get something scheduled?

**Comment by allison0034** [comment]
@rebeccawalsh yes please include me on the invite. Thanks! : )  Peggy also should probably be invited. And Shira as optional. Typically midpoint reviews are done before testing, since the changes you make may have an impact on that.

1.  I am assuming you are referring the the action link. @rtwell are you ok with not using the green action link for sign in? And I just want to clarify you will try to use the [existing pattern ](https://design.va.gov/patterns/benefit-applications)we have for sign in.
2. The little blue line and tool are not currently in the design system so they should be removed. 
3. If these pages you are creating are new then we would like to just do the questions one per page. We want to retire the wizard. If this page already exists and it already has the wizard and you are updating the wizard, the preference would be to redo it in single pages if possible. From your last statement it sounds like you are just pulling in an existing wizard. Maybe we can chat during the meeting with Ryan? If there are constraints and you do not have a choice then you can stick with the wizard.
4. I think Ryan will chat more about this in the meeting.

**Comment by LindaLi123** [comment]
@allison0034 Hello Allison, do we need to do anything else before we can close this ticket?  Changes that we previously discussed has been implemented.

**Comment by allison0034** [comment]
No, thank you
